A66-otel-stats.md

```diff
@@ -0,0 +1,325 @@
+  OpenTelemetryPluginBuilder& SetMeterProvider(shared_ptr<opentelemetry::metrics::MeterProvider> meter_provider);
+  // Enable metric \a metric_name.
+  OpenTelemetryPluginBuilder&  EnableMetric(absl::string_view metric_name);
+  // Disable metric \a metric_name
+  OpenTelemetryPluginBuilder&  DisableMetric(absl::string_view metric_name);
+  // Builds and registers a OpenTelemetry Plugin
+  void BuildAndRegisterGlobal();
+};
+
+```
+
+In the future, additional API might be provided to allow registering the plugin
+for a particular channel or server builder.
+
+### Java
+
+To be filled
+
+### Go
+
+To be filled
+
+### Python
+
+To be filled
+
+## Migration from OpenCensus
+
+The following sections show the differences between the gRPC OpenCensus spec and
+the proposed gRPC OpenTelemetry spec and the mapping of metrics between the two.
+It also presents metrics present in OpenCensus spec that do not map to a metric
+in the OpenTelemetry spec at present. Two migration strategies are also proposed
+for customers who are satisfied with the stats coverage provided by the current
+OpenTelemetry spec.
+
+### Metric Schema Comparison
+
+#### Differences from gRPC OpenCensus Spec
+
+*   OpenTelemetry instrument names don’t allow ‘/’ so we use ‘.’ as the
+    separator. We also get rid of the “.io” suffix in “grpc.io” as it doesn’t
+    seem to add any value and is consistent with other names in the metrics spec
+    from OpenTelemetry.
+*   We also use this opportunity to resolve ambiguities from the gRPC OpenCensus
+    spec (detailed below).
+*   OpenTelemetry has attributes similar to tags in OpenCensus, and the
+    OpenCensus tag names already seem to match the OpenTelemetry spec - except
+    for ‘\_’ vs ‘.’ for namespaces. So we just replace the ‘\_’ with ‘.’. Note
+    that the 'client' and 'server' distinction has also been removed since it
+    does not add any benefit.
+    *   grpc_client_method -> grpc.method
+    *   grpc_client_status -> grpc.status
+    *   grpc_server_method -> grpc.method
+    *   grpc_server_status -> grpc.status
+*   Two new attributes have been added.
+    *   grpc.target - Added on client metrics
+    *   grpc.authority - Added on server metrics
+*   Latency metrics in the OpenTelemetry spec use the recommended `s` unit
+    instead of `ms`.
+
+#### Metrics with Corresponding Equivalent
+
+The following OpenCensus metrics have an equivalent in the OpenTelemetry spec
+(with the above noted differences) allowing for receivers of the telemetry data
+to join the views from the two metrics for continuity.
+
+gRPC OpenCensus                  | gRPC OpenTelemetry
+-------------------------------- | ---------------------------------------------
+grpc.io/client/started_rpcs      | grpc.client.attempt.started
+grpc.io/client/completed_rpcs    | (Derivable from grpc.client.attempt.duration)
+grpc.io/client/roundtrip_latency | grpc.client.attempt.duration
```

Comment:
is there a particular reason for this naming convention you're using? it seems like if client metrics are always `grpc.client.attempt.*` and server metrics are always `grpc.server.call.*`, that dimension (`attempt`/`call`) is unnecessary.

Reply:
Yeah, this gRFC doesn't list all of them, but there are potentially many more metrics we would want, for example - 
1) `grpc.client.call.duration`
2) `grpc.client.method.sent_messages` \ `grpc.client.method.rcvd_messages`
3) `grpc.client.connection.opened`
4) `grpc.client.http.pings_sent`